src/points/PointsDiscoverCard.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import { PointsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import LogoHeart from 'src/icons/LogoHeart'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { pointsBalanceSelector } from 'src/points/selectors'
+import { useSelector } from 'src/redux/hooks'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+}
+
+export default function PointsDiscoverCard({ style }: Props) {
+  const showPoints = getFeatureGate(StatsigFeatureGates.SHOW_POINTS)
+
+  const { t } = useTranslation()
+  const pointsBalance = useSelector(pointsBalanceSelector)
+
+  const handlePress = () => {
+    ValoraAnalytics.track(PointsEvents.points_screen_open)
```

Comment:
nit: can this event name be something like `points_discover_press`? If we want the points screen to ever have more entry points, then reusing `points_screen_open` from those new locations, would be ambiguous as to where the user entered from, unless we add on an e.g. `fromScreen` property or something similar.